src/main/java/com/amazon/opendistroforelasticsearch/sql/rewriter/matchtoterm/TermFieldRewriter.java

```diff
@@ -147,19 +149,27 @@ public boolean visit(SQLIdentifierExpr expr) {
 
     public void collect(SQLTableSource tableSource, Map<String, String> indexToType,  Map<String, String> aliases) {
         if (tableSource instanceof SQLExprTableSource) {
+
             String table = null;
```

Comment:
np: Should this be tableName?

Reply:
Apologies. Lets keep it `table` as before

Reply:
I think your comment was pretty reasonable, since it's not the table object, but the name of the table. Naming is very important since it's one of the first if not the first thing that affects readability.